src/types.tsx

```diff
@@ -26,5 +26,5 @@ export interface AllQuestionFormat{
   //external_services:QuestionFormat;
   //survey:QuestionFormat;
 
-export const QuestionContext = React.createContext<AllQuestionFormat | {} >({})
+export const QuestionContext = React.createContext<any>({})
```

Comment:
There must be a way we can make the interface work. I will deff take a look later tonight